.storybook/preview.tsx

```diff
@@ -8,14 +8,6 @@ const preview: Preview = {
     layout: 'fullscreen',
   },
 
-  decorators: [
-    (Story) => (
-      <React.Suspense fallback={null}>
-        <Story />
-      </React.Suspense>
```

Comment:
@CodyJasonBennett | @drcmda: I think this is not required anymore since r3f now has this

Reply:
I believe this was here since Storybook would crash prior if something suspended into its component tree.

Reply:
Let's try removing it.